test/playlist-maker.test.js

```diff
@@ -34,12 +34,6 @@ const videoList = [{
     type: 'video/mp4'
   }],
   poster: 'http://media.w3.org/2010/05/video/poster.png'
-}, {
```

Comment:
Most of the changes to this file are, in fact, a revert of the changes introduced in #115